### PR TITLE
LPAL-149 run coverage in build

### PR DESCRIPTION
**Pull request by nickdavis2001**
## Purpose

_Run coverage on PHP code in the build so we can get more info on how much the code is covered by the unit tests_

## Approach

_Use the coverage option on phpunit in the workflow_

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] I have added mandatory tags to terraformed resources, where possible
* [ ] If I have a new OPG component dependency, I have updated the `metadata.json` with the repo location.
* [ ] If I added a package.json or composer.json, I also made sure this is included in the script in `.github/workflows/dependabot-update.yml`
* [ ] The product team have tested these changes

### CODE REVIEW

**Comment by townxelliot** [comment]
Can you please squash these commits down into one, as you only touch one file? Thanks.

File: .github/workflows/phpunit.yml [hunk 1]
```diff
@@ -85,4 +81,12 @@ jobs:
       - name: Composer install
         run: composer install --prefer-dist --optimize-autoloader --no-suggest --no-interaction
       - name: Run phpunit
-        run: ./vendor/bin/phpunit
+        #run: XDEBUG_MODE=coverage php ./vendor/bin/phpunit --coverage-html ./coverage-html
```
Comment (townxelliot) [inline .github/workflows/phpunit.yml:84]: Remove this line please.

**Review by townxelliot — CHANGES_REQUESTED** [review]

File: .github/workflows/phpunit.yml [hunk 2]
```diff
@@ -73,10 +73,6 @@ jobs:
           role-to-assume: arn:aws:iam::311462405659:role/opg-lpa-ci
           role-duration-seconds: 1800
           role-session-name: OPGMakeaLPAECRGithubAction
-      - name: Setup PHP
```
[the remark this answers is not in the document]
Comment (nickdavis2001) [reply .github/workflows/phpunit.yml]: OK reinstated, that pins it at 8.1 which we still need for now

File: .github/workflows/phpunit.yml (hunk 1 above)
Comment (nickdavis2001) [reply .github/workflows/phpunit.yml]: Done

**Review by townxelliot — APPROVED** [review]

**Pull request merged**